README.md

```diff
@@ -3,17 +3,19 @@ services: app-service, key-vault
 platforms: python
```

Comment:
Change this to 

platforms -> languages
services -> products

Make sure all samples follow that as well.